### PR TITLE
Move rootfs jpl files from kernelci-core/jenkins

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Move rootfs-builder.jpl and rootfs-trigger.jpl from
kernelci-core and update jobs.groovy file.

Signed-off-by: Lakshmipathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

File: jobs.groovy [hunk 1]
```diff
@@ -200,13 +200,13 @@ pipelineJob('rootfs-build-trigger') {
       lightweight(true)
       scm {
         git {
-          branch(KCI_CORE_BRANCH)
+          branch(KCI_JENKINS_BRANCH)
           remote {
-            url(KCI_CORE_URL)
+            url(KCI_JENKINS_URL)
```
Comment (gctucker) [inline jobs.groovy:205]: `KCI_JENKINS_URL` and `KCI_JENKINS_BRANCH` are not defined.  They need to be added to the list of job parameters, and any default values need to be defined at the top of this file.

**Comment by gctucker** [comment]
Thanks @Lakshmipathi, the job-dsl now worked on staging:
https://bot.staging.kernelci.org/job/job-dsl/288/

Can you please check the jobs were created with the correct settings and run them to check they work as expected, using the Jenkins job files from this repository?
https://bot.staging.kernelci.org/job/rootfs-build-trigger/
https://bot.staging.kernelci.org/job/rootfs-builder/

**Comment by Lakshmipathi** [comment]
@gctucker  Verified rootfs job have proper parameters. tested them and verified logs.
https://bot.staging.kernelci.org/job/rootfs-build-trigger/1/
https://bot.staging.kernelci.org/job/rootfs-builder/1/

**Comment by gctucker** [comment]
Thanks - however that only resulted in one rootfs to get built.  I've started another trigger job to build all the variants.

**Pull request merged**